setup.cfg

```diff
@@ -0,0 +1,25 @@
+[metadata]
+name = gdc-rnaseq-tools
+description = RNASeq tools
+url = https://github.com/NCI-GDC/gdc-rnaseq-tool
+author = Charles Czysz
+author_email = czysz@uchicago.edu
+python_requires = >=3.6
+license_files = LICENSE
+long_description = file:README.md
+long_description_content_type = text/markdown
+classifiers =
+        License :: OSI Approved :: Apache Software License
+
+[options]
+install_requires =
```

Comment:
Updated location to declare install dependencies.
Development dependencies are also now specified in `dev-requirements.in`, but those shouldn't change too much.